app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.android.app.devoptions
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+/** Activity for Developer Options. */
+class DeveloperOptionsActivity : InjectableAppCompatActivity() {
```

Comment:
@Sarthak2601 Was this suppose to be a separate module or just separate package?

Reply:
Earlier it was supposed to be a separate module, but now we've shifted it to separate package approach.

Reply:
Okay.